cleanlab_studio/internal/upload_helpers.py

```diff
@@ -39,6 +39,10 @@ def upload_dataset(
         if modality is not None:
             schema["metadata"]["modality"] = modality
         if id_column is not None:
+            if id_column not in schema["fields"]:
```

Comment:
should we also add this check for `label_column`, `text_column`, etc.?

Reply:
During upload, it seems that only the `id_column` is used, because other columns are inferred.

During project creation, label column seems already to be handled [here](https://github.com/cleanlab/cleanlab-studio/blob/a7aebfb3da81b62b857dd88a480151ee64639cca/cleanlab_studio/studio/studio.py#L193), text column [here](https://github.com/cleanlab/cleanlab-studio/blob/a7aebfb3da81b62b857dd88a480151ee64639cca/cleanlab_studio/studio/studio.py#L212), and that code block also handled a few other cases.

Reply:
Ohh idk why i thought this was the project creation code